docs/source/tutorials/nemo.rst

```diff
@@ -1,8 +1,152 @@
+    BIND_OPTS="${BIND_OPTS}/usr/lib64/liblustreapi.so.1:/opt/lib64/liblustreapi.so.1,"
+    BIND_OPTS="${BIND_OPTS}/usr/lib64/liblustreapi.so.1.0.0:/opt/lib64/liblustreapi.so.1.0.0,"
+    BIND_OPTS="${BIND_OPTS}/opt/cray/libfabric/1.11.0.4.71/lib64/:/opt/fabric,"
+    BIND_OPTS="${BIND_OPTS}/usr/lib64/libpals.a:/opt/lib64/libpals.a,"
+    BIND_OPTS="${BIND_OPTS}/usr/lib64/libpals.so:/opt/lib64/libpals.so,"
+    BIND_OPTS="${BIND_OPTS}/usr/lib64/libpals.so.0:/opt/lib64/libpals.so.0,"
+    BIND_OPTS="${BIND_OPTS}/usr/lib64/libpals.so.0.0.0:/opt/lib64/libpals.so.0.0.0,"
+    BIND_OPTS="${BIND_OPTS}/opt/cray/pe/mpich/8.1.9/ofi/gnu/9.1:/opt/mpi/install/,"
+    BIND_OPTS="${BIND_OPTS}/opt/cray/pe/pmi/6.0.13/lib,"
+    BIND_OPTS="${BIND_OPTS}/var/spool/slurmd/mpi_cray_shasta"
+    
+    export SINGULARITYENV_LD_LIBRARY_PATH=/opt/hdf5/install/lib:$SINGULARITYENV_LD_LIBRARY_PATH
+    export SINGULARITYENV_LD_LIBRARY_PATH=/opt/mpi/install/lib-abi-mpich:$SINGULARITYENV_LD_LIBRARY_PATH
+    export SINGULARITYENV_LD_LIBRARY_PATH=/.singularity.d/libs:$SINGULARITYENV_LD_LIBRARY_PATH
+    export SINGULARITYENV_LD_LIBRARY_PATH=/opt/cray/pe/pmi/6.0.13/lib:$SINGULARITYENV_LD_LIBRARY_PATH
+    export SINGULARITYENV_LD_LIBRARY_PATH=/opt/lib64:$SINGULARITYENV_LD_LIBRARY_PATH
+    export SINGULARITYENV_LD_LIBRARY_PATH=/opt/fabric:$SINGULARITYENV_LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH=/opt/hdf5/install/lib
+    export OMP_NUM_THREADS=1
+    
+    singularity run ${BIND_OPTS} --home [RUNING-DIR] nemo.sif [NEMO-XIOS]
+
+The CoNES `documentation <https://cones.readthedocs.io/en/latest/?badge=latest>`__ gives additional 
+information on how to run NEMO singularity containers.
+
+
+Example output from NEMO-ERSEM
+##############################
+
+To visualise NEMO output, we recommend using `nctoolkit <https://github.com/pmlmodelling/nctoolkit>`__.
+A basic example of how to use `nctoolkit` is given below. Within the 
+`documentation <https://nctoolkit.readthedocs.io/en/latest/>`__ of `nctoolkit` one will find additional
+example uses.
+
+.. code-block:: python
+    :caption: Example plotting script using `nctoolkit`, user is required to change `[PATH_TO_NETCDF_FILE]` and 
+            `[VARIABLE]` to the exact location of the NEMO output and the variable to be plotted, respectively.
+
+    import nctoolkit as nc
+    
+    # path to netCDF file
+    ff = [PATH_TO_NETCDF_FILE]
+    ds = nc.open_data(ff)
+
+    # removes colours from land values
+    ds.set_missing(0)
+
+    # draws the outline of the land
+    ds.fix_nemo_ersem_grid()
+
+    # selects the final timestep for plotting
+    ds.select(time=[len(ds.times)-1])
+
+    # plots the variable. Note, the colour bar will change for each plot since autoscale
+    # is set to False
+    plot = ds.plot([VARIABLE], autoscale=False)
+
+The following plots show the surface distributions of a subset of variables from the NEMO-ERSEM simulation on the AMM7 domain.
+
```

Comment:
1. Suggest to provide date for the snapshots.
2. Not essential, but max clim for nitrate and DIC could be trimmed.
3. Suggest to add in a small font: "Figures provided for illustrative purposes", i.e. a warning for them not to be used to judge model performance.

Reply:
1. Can do
2. Do you mean to cap the clim and thus all values above a certain value will be the same?
3. would this be on all figures or just at the top of the figures?